scene/gui/tree.cpp

```diff
@@ -4231,7 +4234,8 @@ void Tree::_notification(int p_what) {
 			cache.rtl = is_layout_rtl();
 
 			if (root && get_size().x > 0 && get_size().y > 0) {
-				draw_item(Point2(), draw_ofs, draw_size, root);
+				int self_height;
```

Comment:
Should we initialize this value? I understand it's discarded anyway, which is kind of meh as well, just not sure if not having it correctly initialize can lead to problems. Also, a comment would be handy that we have no use for it.